includes/Modules/Sign_In_With_Google.php

```diff
@@ -388,7 +388,8 @@ private function render_signinwithgoogle_woocommerce() {
 	 *                rendered the code to replace buttons.
 	 */
 	private function render_signinwithgoogle() {
-		$is_wp_login          = is_login();
+		// `is_login()` isn't available until WP 6.1.
+		$is_wp_login          = false !== stripos( wp_login_url(), $_SERVER['SCRIPT_NAME'] ?? '' ); // phpcs:ignore WordPress.Security.ValidatedSanitizedInput
```

Comment:
This test was already failing (see #10266) so we can fix it here rather than merge this with broken tests.